setup.py

```diff
@@ -42,9 +42,9 @@ def get_version(version_file):
 
 
 setup(
-    name="example",
+    name="pe_reports",
     # Versions should comply with PEP440
-    version=get_version("src/example/_version.py"),
+    version=get_version("src/pe_reports/_version.py"),
```

Comment:
The `description` on the next line should be updated from the skeleton default text.

Reply:
Resolved in e294e62